ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -543,7 +543,15 @@ class MistakeCollector {
     // wipe then (to make post-wipe deaths more obvious), however this
     // requires making liveList be able to insert items in a sorted
     // manner instead of just being append only.
-    this.OnFullMistakeText('wipe', null, 'Party Wipe');
+    const wipeLang = {
```

Comment:
Can you put this at the top with the `kEarlyPullText` / `kLatePullText` etc, so it isn't hidden down in this function?